metadata_layer.py

```diff
@@ -463,32 +800,24 @@ def __init__(
             contributors=None,
             measurements=None,
             links=None,
-            formats=None,
-            rights_uri=None,
             last_update_time=None,
-            circulation=None,
+            #circulation=None,
     ):
-        # data_source is where the data comes from.
+        # data_source is where the data comes from (e.g. overdrive, metadata wrangler, admin interface), 
+        # and not where the lending licenses are coming from.
```

Comment:
This is no longer accurate. If there is any CirculationData to be had here, it comes from the same DataSource as the Metadata.